CHANGELOG.md

```diff
@@ -4,13 +4,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-layout** Updated main content spacing
```

Comment:
I'm not very knowledgeable about the potential side effects of this change but 'minor' seems good to me.